TicTacToe/Forc.toml

```diff
@@ -0,0 +1,2 @@
+[workspace]
+members = ["./tictactoe-contract"]
```

Comment:
Do we need this file?

Reply:
Same as the comment above. While this project just has one contract, some have more such as the AMM.